routes/project/user.rb

```diff
@@ -171,14 +171,36 @@ class Clover
       r.post "policy/managed" do
         authorize("Project:user", @project.id)
 
-        invitation_policies = r.params["invitation_policies"] || {}
-        allowed_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:add").select_hash(:name, Sequel.as(true, :v))
-        invitation_policies.each do |email, policy|
-          if allowed_tags[policy]
-            @project.invitations.find { _1.email == email }&.update(policy: policy)
+        DB.transaction do
+          allowed_add_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:add").all.to_h { [_1.name, _1] }
+          allowed_remove_tags = dataset_authorize(@project.subject_tags_dataset, "SubjectTag:remove").all.to_h { [_1.name, _1] }
+
+          user_policies = r.params["user_policies"] || {}
+          user_policies.each do |ubid, policy|
+            user = Account.from_ubid(ubid)
+            existing_tags = user.subject_tags_dataset.where(project_id: @project.id).select_map(:name)
```

Comment:
We need to take no action on the user here if the user is in more than one tag.

Reply:
I added this check